resources/migrations/056_update_migrations.yaml

```diff
@@ -169,7 +169,322 @@ databaseChangeLog:
+                    nullable: false
+              - column:
+                  name: config
+                  remarks: App configuration JSON
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
+                  name: entity_id
+                  remarks: SHA hash for the entity
+                  type: char(21)
+                  constraints:
+                    nullable: true
+              - column:
+                  remarks: The timestamp of when the definition was created
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+      rollback:
+        - dropTable:
+            tableName: data_app_definition
+
+  - changeSet:
+      id: v56.2025-07-03T16:02:00
+      author: qnkhuat
+      comment: Add index on data_app_definition.app_id
+      preConditions:
+        - onFail: MARK_RAN
+        - tableExists:
+            tableName: data_app_definition
+        - not:
+            - indexExists:
+                tableName: data_app_definition
+                indexName: idx_data_app_definition_app_id
+      changes:
+        - createIndex:
+            tableName: data_app_definition
+            indexName: idx_data_app_definition_app_id
+            columns:
+              - column:
+                  name: app_id
+      rollback: #no op, will be deleted with the table
+
+  - changeSet:
+      id: v56.2025-07-03T16:02:10
+      author: qnkhuat
+      comment: index data_app_definition.app_id and revision_number
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            - indexExists:
+                tableName: data_app_definition
+                indexName: idx_data_app_definition_app_id_revision_number
+      changes:
+        - createIndex:
+            tableName: data_app_definition
+            indexName: idx_data_data_app_definition_id_revision_number
+            columns:
+              - column:
+                  name: app_id
+              - column:
+                  name: revision_number
+                  descending: true
+      rollback: #no op, will be deleted with the table
+
+  - changeSet:
+      id: v56.2025-07-03T16:02:20
+      author: qnkhuat
+      comment: Add unique constraint on data_app_definition (revision_number, app_id)
+      preConditions:
+        - onFail: MARK_RAN
+        - tableExists:
+            tableName: data_app_definition
+        - not:
+            - uniqueConstraintExists:
+                tableName: data_app_definition
+                constraintName: idx_data_app_definition_revision_number_app_id
+      changes:
+        - addUniqueConstraint:
+            tableName: data_app_definition
+            constraintName: idx_data_app_definition_revision_number_app_id
+            columnNames: revision_number, app_id
```

Comment:
Why is the order flipped this time compared to the other index?

This is another example of redundancy - if we have the unique index we get the look-up for free as well.